server/src/main/java/com/kritsit/casetracker/server/datalayer/VehicleRepository.java

```diff
@@ -44,4 +44,23 @@ public VehicleRepository(IPersistenceService db){
             throw new RowToModelParseException("Error retrieving vehicles from database for defendant name: " + defendant.getName());
         }
     }
+
+	@Override
+	public void insertVehicles(Vehicle vehicle, Defendant defendant) {
+		try{
+			logger.info("Inserting a vehicle for defendant {}", defendant.getName());
+			String sql = "INSERT INTO vehicles VALUES ('"
+					+vehicle.getRegistration()+"', '"
+					+defendant.getId()+"', '"
```

Comment:
I know you know, but I'm going to put this here anyway for future reference - `defendant.getId()` returns the identity number of the defendant (Social security or whatever it is called in various countries), however as this can be null (E.g. when prosecuting companies), defendants have a indexID field in the DB. This value references that to form a foreign key.